code/api/src/modules/productDelivery/types.js

```diff
@@ -6,13 +6,13 @@ import { ProductType } from '../product/types'
 
 // Product Delivery Type
 const ProductDeliveryType = new GraphQLObjectType({
-  name: 'product delivery',
+  name: 'ProductDelivery',
   description: 'Product Delivery Type',
 
   fields: () => ({
     id: { type: GraphQLInt },
     product: { type: ProductType },
```

Comment:
Is it possible to add a name for the product. So that FE can display that for the user?

Reply:
nvm. I found it in product!